temporal-normalizer/src/main/java/conf/heideltime_config.props

```diff
@@ -0,0 +1,73 @@
+################################
```

Comment:
Please create a Configurator class that sets these options programmatically. See https://github.com/IllinoisCogComp/illinois-cogcomp-nlp/blob/master/pos/src/main/java/edu/illinois/cs/cogcomp/pos/POSConfigurator.java
for an example.